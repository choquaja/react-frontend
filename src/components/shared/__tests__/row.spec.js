import React from 'react';
import { shallow } from 'enzyme';
import Row from '../row';

describe('row', () => {
  it('renders without crashing', () => {
    shallow(<Row />);
  });

  it('renders with justify prop', () => {
    const component = shallow(<Row justify />);
    expect(component).toMatchSnapshot();
  });

  it('renders with the align prop', () => {
    const component = shallow(<Row align />);
    expect(component).toMatchSnapshot();
  });
});
