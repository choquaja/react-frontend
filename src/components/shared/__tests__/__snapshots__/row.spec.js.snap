// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`row renders with justify prop 1`] = `
<div
  className="row__Row-kCedir bEVabT"
/>
`;

exports[`row renders with the align prop 1`] = `
<div
  className="row__Row-kCedir kuhlph"
/>
`;
